### PR TITLE
very simple dev strategy

### PR DESCRIPTION
**Pull request by unmeshvrije**

### CODE REVIEW

File: Planning/Developement-strategy.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Things todo:
```
Comment (zaraana) [inline Planning/Developement-strategy.md]: This title does not describe dev strategy

File: Planning/Developement-strategy.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+* Make a project board
+* Write small issues to work on
```
Comment (zaraana) [inline Planning/Developement-strategy.md]: How do you define small issues?

**Review by zaraana — CHANGES_REQUESTED** [review]

File: Planning/Developement-strategy.md (hunk 2 above)
Comment (kilicf) [reply Planning/Developement-strategy.md]: I agree with @zaraana 👍

**Review by kilicf — CHANGES_REQUESTED** [review]

**Review by zaraana — APPROVED** [review]
Great Job @unmeshvrije  You are the best :)

**Review by kilicf — APPROVED** [review]

**Pull request merged**